pyproject.toml

```diff
@@ -1,39 +1,38 @@
-[project]
```

Comment:
you are missing [project] section in the pyproject.toml , that's why CI pipeline fails